posthog/api/plugin.py

```diff
@@ -354,15 +352,30 @@ def job(self, request: request.Request, **kwargs):
         job_payload = job.get("payload", {})
         job_op = job.get("operation", "start")
 
-        celery_app.send_task(
-            name="posthog.tasks.plugins.plugin_job",
-            queue=settings.PLUGINS_CELERY_QUEUE,
-            args=[self.team.pk, plugin_config_id, job_type, job_op, job_payload],
+        payload_json = json.dumps(
+            {
+                "type": job_type,
+                "payload": job_payload | {"$operation": job_op},
+                "pluginConfigId": plugin_config_id,
+                "pluginConfigTeam": self.team.pk,
+            }
         )
+        sql = f"SELECT graphile_worker.add_job('pluginJob', '{payload_json}')"
+        execute_postgres(sql)
```

Comment:
Let's use SQL interpolation instead of f-strings.

Reply:
I think it's only possible for the whole json, unless you know how I can make that work for separate params inside:
```
Exception: Failed to execute postgres sql=SELECT graphile_worker.add_job('pluginJob', '{"type": %s, "payload": %s, "pluginConfigId": %s, "pluginConfigTeam": %s}'),
params=['List all subscribers', {'$operation': 'start'}, 2, 1],
exception=syntax error at or near "List"
LINE 1: ...T graphile_worker.add_job('pluginJob', '{"type": 'List all s...
```

Reply:
Yep, meant just that